main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -317,4 +317,16 @@ object LexerError {
          |""".stripMargin
     }
   }
+
+  /** Returns an ASCII printable version of `c`. */
+  private def showChar(c: Char): String = {
```

Comment:
I'm not sure where to put this function. `Util.Char` maybe?

Reply:
OK to leave it.